src/com/estafet/learning/US008/Invoice.java

```diff
@@ -0,0 +1,27 @@
+package com.estafet.learning.US008;
+
+import java.util.Objects;
+
+public abstract class Invoice extends Order{
```

Comment:
according to task definition inheritance should be the following:
- `TradeIvoice` extends `Invoice`
- `TradeOrder` extends `Order`

Reply:
changed.